Modules/_gdbmmodule.c

```diff
@@ -122,12 +122,6 @@ newgdbmobject(_gdbm_state *state, const char *file, int flags, int mode)
 }
 
 /* Methods */
-static int
-gdbm_traverse(PyObject *op, visitproc visit, void *arg)
-{
-    Py_VISIT(Py_TYPE(op));
-    return 0;
-}
 
```

Comment:
General point -- check on whitespace, the deleted function here should also have its trailing blank line deleted 
```suggestion
```